scan-apk/action.yaml

```diff
@@ -70,7 +70,7 @@ runs:
         tag: localhost:1234/apk-scan
 
     - id: grype-scan
-      uses: anchore/scan-action@ecfd0e98932e57ea8f68f29c4f418fc41a8194db
+      uses: anchore/scan-action@24fd7c9060f3c96848dd1929fac8d796fb5ae4b4
```

Comment:
This change seems somewhat unrelated to the rest of the PR?

Reply:
Yes, it seems unrelated but without this minor bump [`v3.2.5`](https://github.com/anchore/scan-action/releases/tag/v3.2.5) -> [`v3.3.6`](https://github.com/anchore/scan-action/releases/tag/v3.3.6) not all `::set-ouput` warnings go away, because used inside actual pinned version of `anchore/scan-action`. Maybe a different version can be selected, I have chosen this cause it's the last one released and surely in there `::set-output` warnings are fixed